pages/404.jsx

```diff
@@ -45,6 +46,6 @@ export default function PageNotFound() {
 
         <Link href="/">&lt; Return whence you came.</Link>
       </main>
-    </div>
+    </Layout>
```

Comment:
We're going to have multiple `main` elements here, since the `Layout` already includes it.